ftxwebsocket/client.py

```diff
@@ -7,20 +7,21 @@
 from typing import DefaultDict, Deque, List, Dict, Tuple, Optional
 from gevent.event import Event
 
-from websocket.websocket_manager import WebsocketManager
+from ftxwebsocket.websocket_manager import WebsocketManager
```

Comment:
Rename `websocket` to `ftxwebsocket` (and `rest` to `ftxrest` for consistency) to avoid package naming conflicts with the `websocket-client` PyPI package, which populates the `websocket` namespace.